database/slashing.go

```diff
@@ -71,3 +71,20 @@ WHERE slashing_params.height <= excluded.height`
 
 	return nil
 }
+
+// GetValidatorTombstonedStatus gets the validator latest tombstoned status from the database
+func (db *Db) GetValidatorTombstonedStatus(valConsAddress string) (bool, error) {
```

Comment:
I only have question, which I don't remember the answer to: if we're querying this value, when are we updating it? What would happen is a validator is jailed or tombstoned in a new block? Is the value updated somewhere else or what?

Reply:
@ryuash Can we have some more context on why this is needed/required/preferable?

Reply:
Otherwise we just get the `tombstone status` from the `validator_signing_info` table, gql query be something like this
```gql
query MyQuery {
  validator_status {
    height
    jailed
    status
    validator_address
    validator {
      validator_signing_infos {
        tombstoned
      }
    }
  }
}
```

in this case I will remove the tomstoned column from `validator_status` table.

would this be reasonable for you @yayay927 ?

Reply:
I remember this was requested months back. I no longer remember the purpose tbh

Reply:
then I think we can remove this part of codes because it keeps causing "transport is closing" error while calling the node :-/ we should then alternatively JOIN the `validator_signing_info` table for `tombstone` status.

Reply:
OH SORRY I REMEMBER NOW. Yes this was requested during the first wave of tombstones validators as we had no way of seeing who was tombstoned in bdjuno at the time. 

It's currently being used in the frontend so a migration would be needed if you're making any changes

Reply:
Honestly if BD is fine as it is right now, I would avoid updating anything

Reply:
I see, but we actually get a lot of errors like this while running staking module
```
error="error while getting validator signing info: 
rpc error: code = Unavailable desc = transport is closing" height=621512 module=staking
```

saving tombstone twice is weird no? when we can just join the tables